lib/Stan_math_code_gen.ml

```diff
@@ -19,7 +19,7 @@
 open Core_kernel
 open Mir
 open Fmt
-open Expression_gen
+module ExprGen = Expression_gen
```

Comment:
Why shouldn't we just open it? It's basically only in a separately file because that's how it was added - it's very much integral with this module. 

Separately (in another PR) I think we might want to move the pretty printing of the MIR out of that module and just keep the types in there, which will help with name clashes as a byproduct.

Reply:
This is mainly habit - I usually try and avoid opening modules altogether but I take your point on them having the same purpose. I'll un-qualify it and add and look at merging the files in #99.